src/shared/components/query/GeneSetSelector.tsx

```diff
@@ -40,16 +41,29 @@ export default class GeneSetSelector extends QueryStoreComponent<GeneSetSelector
 
 	@computed get geneListOptions()
 	{
-		return [
-			{
-				label: 'User-defined List',
-				value: ''
-			},
-			...gene_lists.map(item => ({
-				label: `${item.id} (${item.genes.length} genes)`,
-				value: item.genes.join(' ')
-			}))
-		];
+	    if (AppConfig.querySetsOfGenes == null || AppConfig.querySetsOfGenes == undefined) {
+	        return [
+                    {
+                        label: 'User-defined List',
+                        value: ''
+                    },
+                    ...gene_lists.map(item => ({
+                        label: `${item.id} (${item.genes.length} genes)`,
+                        value: item.genes.join(' ')
+                    }))
+                ];
+	    } else {
+	        return [
+	                {
+	                    label: 'User-defined List',
+	                    value: ''
+	                },
+	                ...AppConfig.querySetsOfGenes.map(item => ({
```

Comment:
Seems like these 2 cases are copy/paste of each other besides the list name, can be improved by creating a local variable for the list in this function, something like:
```
let geneList = gene_lists;
if (AppConfig. querySetsOfGenes) {
    geneList = AppConfig. querySetsOfGenes;
}

return [
	{
		label: 'User-defined List',
		value: ''
	},
	...geneList.map(item => ({
		label: `${item.id} (${item.genes.length} genes)`,
		value: item.genes.join(' ')
	}))
];